README.md

```diff
@@ -1,52 +1,64 @@
-# Docker-OSX
-## [Follow @sickcodes on Twitter](https://twitter.com/sickcodes)
+# Docker-OSX 
 
-![Running mac osx in a docker container](/running-mac-inside-docker-qemu.png?raw=true "OSX KVM DOCKER")
+![Running Mac OS X in a Docker container](/running-mac-inside-docker-qemu.png?raw=true "OSX KVM DOCKER")
 
-Run Mac in a Docker container! Run near native OSX-KVM in Docker! X11 Forwarding! iMessage security research!
+Run OSX-KVM in Docker with near-native performance! X11 Forwarding! iMessage security research!
```

Comment:
Run Mac using OSX-KVM in Docker with near-native performance! X11 Forwarding! iMessage security research!

Reply:
Perhaps also add

```
## Docker-OSX Repository:

[https://github.com/sickcodes/Docker-OSX](https://github.com/sickcodes/Docker-OSX)
```

Because inside hub.docker.com, there is no obvious link back to this place

Reply:
> Because inside hub.docker.com, there is no obvious link back to this place

Hyperlinking links isn't necessary and superfluous, the bracket/parenthesis trick is only used for hyperlinking text. If you look at my fork, you'll notice that the link exists.

Reply:
> Run Mac using OSX-KVM in Docker with near-native performance! X11 Forwarding! iMessage security research!

Will fix that after the other replies get addressed to save time.